nx-X11/lib/Imakefile

```diff
@@ -130,22 +130,6 @@ RANDRLIBDIR = Xrandr
 XEVIELIBDIR = Xevie
 #endif
 
-#if BuildXfixesLibrary
-XFIXESLIBDIR = Xfixes
-#endif
-
-#if BuildDamageLibrary
-DAMAGELIBDIR = Xdamage
-#endif
-
-#if BuildCompositeLibrary
-COMPOSITELIBDIR = Xcomposite
-#endif
-
-#if BuildXcursorLibrary
-XCURSORLIBDIR = Xcursor
-#endif
```

Comment:
Why removing `BuildXcursorLibrary` here? We never shipped this, but still? This should probably not be part of this PR but a unique one?